src/components/panel/panel.spec.js

```diff
@@ -1055,10 +1056,55 @@ describe('$mdPanel', function() {
         pressEscape();
 
         expect(myButton).toBeFocused();
+
       });
     });
   });
 
+  describe('postClose logic:', function() {
+      it('postClose when provided', function () {
+        var obj = {
+          postClose: function() {},
```

Comment:
Why do you need to make an object here? Does it suffice to just create a spy, then pass it to the config?

```
var onCloseSuccess = jasmine.createSpy('onCloseSuccess');
var config = angular.extend({  'postClose': onCloseSuccess });

openPanel(config);
closePanel();

expect(onCloseSuccess).toHaveBeenCalledWith(panelRef, undefined);
```